public/diagnostic.js

```diff
@@ -23,3 +23,15 @@
 //  -  Order 1 : 2 hats ($5 each)
 //  -  Order 2 : 1 desk lamp ($20 each)
 //  -  Order 3 : 3 hand towels ($8 each)
+
+
+const Order = Ember.Object.extend({
+  unitPrice: Ember.computed('amount', function(){
+    return `${this.get('amount')}`;
+  }),
+  quantity: Ember.computed('number of', function(){
+    return `${this.get('number of')}`;
+  }),
+  orderPrice: Ember.computed('total', function(){
+    return `${unitPrice.quantity('total')}` }
+}
```

Comment:
I remember struggling a lot with this one too Stanley. I think you're closer than you think. Review the solution branch and see if you can get a 1 on 1 to go through this. I think walking through and annotating the solution will help a lot 👍 

1/5